src/Devtools/Log.js

```diff
@@ -1,76 +1,66 @@
 /* @flow */
 
-import {Effects, Task, html, forward} from "reflex"
-import {merge, always, nofx} from "../Common/Prelude"
-import {ok, error} from "../Common/Result"
-import * as Runtime from "../Common/Runtime"
-import * as Unknown from "../Common/Unknown"
+import {Effects, html} from 'reflex'
+import {nofx} from '../Common/Prelude'
+import * as Runtime from '../Common/Runtime'
+import * as Unknown from '../Common/Unknown'
 
+import type {Address, DOM} from 'reflex'
 
-import type {Address, Never, DOM, Init, Update, View, AdvancedConfiguration} from "reflex"
-import type {Result} from "../Common/Result"
-import type {URI, ID} from "../Common/Prelude"
-
-
-export type Model <model, action> =
+export type Model <model, action> = // eslint-disable-line no-unused-vars
```

Comment:
Question: Why `eslint-disable-line no-unused-vars` is necessary here ? exports aren't considered unused are they ?